src/plugins/mer/mersettings.cpp

```diff
@@ -215,6 +251,30 @@ void MerSettings::read()
 
     m_environmentFilterFromEnvironment =
         QProcessEnvironment::systemEnvironment().value(Constants::SAILFISH_OS_SDK_ENVIRONMENT_FILTER);
+
+    //! \todo Does not support multiple (different) emulators (not supported at other places anyway).
+
+    // Reading predefined emulators
+    PersistentSettingsReader reader;
+    if (reader.load(deviceModelFileName())) {
```

Comment:
I would prefer returning on error instead of nesting code in blocks. Simply copy the original code without unnecessary changes :)

Reply:
It seems unchanged here, but it looks OK in the parent commit, so you most likely did a mistake during rebase.

Reply:
This code moved to `static QMap<QString, MerEmulatorDeviceModel> deviceModelsRead(const Utils::FileName &fileName)` in the "Add options for user defined device model profiles" commit.